generators/generator-bot-enterprise-assistant/README.md

```diff
@@ -1,68 +1,50 @@
 # @microsoft/generator-bot-enterprise-assistant [![NPM version][npm-image]][npm-url]
 
-This template creates an Enterprise Assistant Bot, comprised of a root bot based on the Core Assistant Bot template, and pre-configured with the Enterprise Calendar Bot and Enterprise People Bot templates as skills. [Learn more](https://aka.ms/EnterpriseAssistant)
+A bot with Azure Language Understanding (LUIS) and common trigger phrases used to direct the conversation flow to help customers accomplish common business tasks. [Learn more](https://aka.ms/EnterpriseAssistant)
 
-Includes support for:
+### Recommended use
 
-- Greeting new and returning users
-- Asking for help
-- Cancelling a dialog
-- Submitting feedback about the bot
-- Error handling in conversations
-- Repeat the previous question
-- Chit chat with QnA Maker ([professional personality](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/chit-chat-knowledge-base?tabs=v1))
-- Disambiguation of NLP results
+- Create a sophisticated bot
+- Customize and extend sophisticated example dialogs, bot logic, language understanding and bot responses
+
+### Included capabilities
 
-Included skills:
 - [Enterprise Calendar Bot](https://aka.ms/EnterpriseCalendarBot)
 - [Enterprise People Bot](https://aka.ms/EnterprisePeopleBot)
+- Greeting new and returning users
+- Bot Framework Orchestrator to direct the conversation flow
+- Chit chat with QnA Maker ([professional personality](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/chit-chat-knowledge-base?tabs=v1))
+- Asking for help
+- Disambiguation of user input (Intents)
```

Comment:
This has a different description of "disambiguation" than Core Assistant

Reply:
changed to use the same language used in core assistant readme